[2025-09-08T17:08:17.409+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:08:17.417+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA backfill__2025-04-12T12:00:00+00:00 [queued]>
[2025-09-08T17:08:17.421+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA backfill__2025-04-12T12:00:00+00:00 [queued]>
[2025-09-08T17:08:17.421+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:08:17.448+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): taskA> on 2025-04-12 12:00:00+00:00
[2025-09-08T17:08:17.459+0000] {standard_task_runner.py:72} INFO - Started process 19815 to run task
[2025-09-08T17:08:17.463+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskA', 'backfill__2025-04-12T12:00:00+00:00', '--job-id', '121', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpxb9r8oau']
[2025-09-08T17:08:17.466+0000] {standard_task_runner.py:105} INFO - Job 121: Subtask taskA
[2025-09-08T17:08:17.495+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskA backfill__2025-04-12T12:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:08:17.542+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2025-04-12T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2025-04-12T12:00:00+00:00'
[2025-09-08T17:08:17.543+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:08:17.554+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-08T17:08:17.554+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK A has executed!']
[2025-09-08T17:08:17.561+0000] {subprocess.py:86} INFO - Output:
[2025-09-08T17:08:17.562+0000] {subprocess.py:93} INFO - TASK A has executed!
[2025-09-08T17:08:17.562+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-08T17:08:17.576+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:08:17.577+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskA, run_id=backfill__2025-04-12T12:00:00+00:00, execution_date=20250412T120000, start_date=20250908T170817, end_date=20250908T170817
[2025-09-08T17:08:17.600+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:08:17.617+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:08:17.621+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
